src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PlacementRequestsTest.kt

```diff
@@ -161,154 +165,110 @@ class PlacementRequestsTest : IntegrationTestBase() {
 
     @ParameterizedTest
     @CsvSource(
-      "/placement-requests/dashboard,nonParole",
-      "/placement-requests/dashboard?isParole=1,parole",
-      "/placement-requests/dashboard?isParole=0,nonParole",
+      "/placement-requests/dashboard,notMatched",
+      "/placement-requests/dashboard?status=notMatched,notMatched",
+      "/placement-requests/dashboard?status=matched,matched",
+      "/placement-requests/dashboard?status=unableToMatch,unableToMatch",
     )
     fun `It returns all the placement requests when the user is a manager`(url: String, type: String) {
```

Comment:
I think this might be better to have this as separate tests, as there's no interdependency between the different types of placement request and there's a lot of nesting.

Reply:
Good call - makes it a bit tider too 👍